content/en/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes.md

```diff
@@ -32,7 +32,7 @@ upgrade the control plane nodes before upgrading your Windows nodes.
 
 1.  From the Windows node, upgrade kubeadm:
 
-    ```powershell
+    ```shell
```

Comment:
:thought_balloon: this isn't right: the code _is_ PowerShell.

Reply:
thanks, @sftim for taking look at this.

If we want to keep the code as Powershell so then we can put the URL (https://dl.k8s.io/{{< param "fullversion" >}}/bin/windows/amd64/...)  into the inverted commas (" ") will resolve this issue.

look like this:

```
# replace v1.24.0 with your desired version
curl.exe -Lo <path-to-kubeadm.exe>  "https://dl.k8s.io/v1.24.0/bin/windows/amd64/kubeadm.exe"
```

Reply:
Is it seems good?

Reply:
Sure. You could try quoting the URL, that looks like it'd be valid PowerShell.

Reply:
> Sure. You could try quoting the URL, that looks like it'd be valid PowerShell.

I will update it soon.

Reply:
I have addressed the comment.
PTAL!

Reply:
You can use single quotes in powershell too. 
The currently changes also work